astro/src/pages/docs/[...overview].astro

```diff
@@ -59,8 +59,7 @@ const entries = currentCat.entries || [];
 const subCategories = currentCat.subcategories;
 ---
 
-<Layout {frontmatter} {navContext} disableTOC={true} excludeFromSearchIndex={true}>
-  {entries.length > 0 && <h1 class="capitalize">{ specialCaps(currentCat.name) } Pages </h1>}
+<Layout title={currentCat.name.charAt(0).toUpperCase() + currentCat.name.slice(1) + " documentation" } {frontmatter} {navContext} disableTOC={true} excludeFromSearchIndex={true}>
```

Comment:
looks fine to me. the `documentation` is maybe redundant given we are in the docs section but I don't feel strongly about it